etl/steps/data/garden/antibiotics/2024-10-23/tracss.meta.yml

```diff
@@ -0,0 +1,90 @@
+# NOTE: To learn more about the fields, hover over their names.
+definitions:
+  common:
+    presentation:
+      topic_tags:
+        - Global Health
+        - Antibiotics
+
+
+# Learn more about the available fields:
+# http://docs.owid.io/projects/etl/architecture/metadata/reference/
+dataset:
+  update_period_days: 365
+
+
+tables:
+  tracss:
+    variables:
+      laws_antimicrobials_terrestrial_2_8_2:
+        title: 2.8.2 - Does the country have laws or regulations on prescription and sale of antimicrobials for terrestrial animal use?
+        unit: ""
+      law_antimicrobials_aquatic_2_8_3:
+        title: 2.8.3 - Does the country have laws or regulations on prescription and sale of antimicrobials for aquatic animals?
+        unit: ""
+      law_antimicrobials_terrestrial_growth_promotion_2_8_5:
+        title: 2.8.5 - Does the country have laws or regulations that prohibits the use of antibiotics for growth promotion in terrestrial animals in the absence of risk analysis?
+        unit: ""
+      monitoring_consumption_human_3_2:
+        title: 3.2 - National monitoring system for consumption and rational use of antimicrobials in human health
+        unit: ""
+        type: ordinal
+        sort:
+            - "No monitoring system"
+            - "National monitoring of antimicrobial sales and antibiotic use in health services"
+            - "National monitoring of antimicrobial sales and some monitoring of antibiotic use at sub-national level"
+            - "Prescribing practices and appropriate antibiotic use monitored in a sample of healthcare settings"
+            - "Regular data collection and reporting"
+      amr_surveillance_human_3_3:
+        title: 3.3 - National surveillance system for antimicrobial resistance (AMR) in humans
+        unit: ""
+        description_key: ["'Common' in this context refers to infections that occur most frequently in both hospitalized and community patients and constitute a majority of indications for antimicrobial treatment (such as urinary tract infections, enteric infections, bloodstream infections, respiratory infections etc). Also indicates that the spectrum is broader than e.g. epidemic-prone (notifiable) infections. Bacterial pathogens causing these infections could be called common pathogens, but just because they cause common infections and so isolated most frequently.‡ Community patients would be in many instances outpatients or those patients within 48 hours of admission in line with GLASS definition§ The term consumption refers to estimates that are derived from aggregated data sources, mainly sales data, and serves as proxy for actual use of antibiotics. Data on antibiotic use refers to estimates derived from individual level data, and may be accompanied by information on patient characteristics and indication of treatment."]
```

Comment:
You could maybe separate this text into more bullet points. There are also some characters to delete: ‡§